homework/index.js

```diff
@@ -30,14 +36,102 @@
     return elem;
   }
 
+  function createOptions(repoNames, select) {
+    createAndAppend('option', select, { text: 'Select module', disabled: 'disabled' });
+    repoNames.sort((a, b) => a.name.localeCompare(b.name));
+    repoNames.forEach((repository, i) => {
+      createAndAppend('option', select, {
+        text: repository.name,
+        value: repository.name,
+      });
+    });
+  }
+
+  function renderRepoInformation(repo, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repo}`;
+    fetchJSON(url, (err, repoInfos) => {
+      if (err) {
+        renderError(err);
+        return;
+      }
+
+      ul.innerHTML = ' ';
+      const li = createAndAppend('li', ul);
+      const table = createAndAppend('table', li, { class: 'infoTable' });
+      const tbody = createAndAppend('tbody', table);
+      const repoName = createAndAppend('tr', tbody);
```

Comment:
You are repeating yourself here four times: create a `tr` with two `td` children. Can you make your code DRYer by creating a function to do this, e.g. a function named `addRow`? You could call it like this:

```js
addRow(tbody, labelText, value);
```

For the first row containing the `a` tag, you would need to do something special, for instance:

```js
const firstRow = addRow(tbody, 'Name', '');
createAndAppend('a', firstRow.lastChild, { href: ...., text: ... });
```